cg/meta/orders/api.py

```diff
@@ -40,10 +40,6 @@ def submit(self, project: OrderType, order_in: OrderIn, user_name: str, user_mai
 
         Main entry point for the class towards interfaces that implements it.
         """
-        try:
-            ORDER_SCHEMES[project].validate(order_in.dict())
```

Comment:
Should this not be validated no more?

Reply:
hey MM!  I don't know, do you thinks it adds enough to be justified?

Reply:
Hi @patrikgrenfeldt !
If we know they are validated in the front end I don't think they need to be validated here. However if they are not I think we should replace the pyschemas with pydantic models